### PR TITLE
docs(logging): update logging guide to reflect logging changes

### LINKED ISSUE #2413: Docs: write "advanced guides -> logging"
I know that bento is in the transition period to 1.0, but I with there was already documentation for the [logging feature](https://docs.bentoml.org/en/latest/guides/logging.html) (especially prediction logging) like there was for [0.13](https://docs.bentoml.org/en/0.13-lts/guides/logging.html).

So this is just a "reminder ticket" that the docs have to be written, and maybe people up vote it :+1: if they think it's important.

### PR DESCRIPTION
**Pull request by sauyon**
Update documentation for loggers to reflect the changes we've made since the alphas.

Should hopefully resolve #2413.

### CODE REVIEW

File: docs/source/guides/logging.rst [hunk 1]
```diff
@@ -2,94 +2,92 @@
...
       access:
           enabled: False
+          # whether to log the size of the request body
           request_content_length: True
+          # whether to log the content type of the request
           request_content_type: True
+          # whether to log the content length of the response
           response_content_length: True
+          # whether to log the content type of the response
           response_content_type: True
```
Comment (aarnphm) [inline docs/source/guides/logging.rst:73]: Having the comments next to the field makes it more readable imo
```suggestion
          request_content_length: True     # whether to log the size of the request body
          request_content_type: True        # whether to log the content type of the request
          response_content_length: True  # whether to log the content length of the response
          response_content_type: True     # whether to log the content type of the response
```

File: docs/source/guides/logging.rst [hunk 2]
```diff
@@ -2,94 +2,92 @@
 Customize Logging
 =================
 
-BentoML provides a powerful and detailed logging pattern out of the box. Request logs for webservices are logged along with requests to each of the model runner services. We use `RichHandler <https://rich.readthedocs.io/en/stable/logging.html>`_ to provide appropriate color coding to the logs in order to make them more easily read.
+BentoML provides a powerful and detailed logging pattern out of the box. Request logs for
```
Comment (parano) [inline docs/source/guides/logging.rst:5]: Should we add a section here about logging out side of the context of model server? like giving user an example of configuring logging before calling `save_model`, similar to this example https://github.com/bentoml/gallery/blob/main/quickstart/train.py#L4-L6

File: docs/source/guides/logging.rst (hunk 1 above)
Comment (sauyon) [reply docs/source/guides/logging.rst:73]: I think this would cause line wrapping, especially in small (e.g. mobile) browsers.

Comment (sauyon) [reply docs/source/guides/logging.rst:77]: I'm open to changing the format though.

File: docs/source/guides/logging.rst [hunk 3]
```diff
@@ -2,94 +2,115 @@
...
+`default python logging configuration <https://docs.python.org/3/howto/logging.html>`_. All BentoML
+logs are logged under the ``"bentoml"`` namespace.
 
 Web Service Request Logging
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+"""""""""""""""""""""""""""
 
-For web requests, logging can be enabled and disabled using the `logging.access` parameter at the top level of the bentofile.yaml.
+For web requests, logging can be enabled and disabled using the `logging.access` parameter at the
+top level of the bentofile.yaml.
```
Comment (parano) [inline docs/source/guides/logging.rst:63]:
```suggestion
top level of the :code:`bentoml_configuration.yml`.
```

File: docs/source/guides/logging.rst [hunk 4]
```diff
@@ -2,94 +2,115 @@
...
 
 Model Runner Request Logging
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+""""""""""""""""""""""""""""
 
-Depending on how you’ve configured BentoML, the webserver may be separated from the model runner. In either case, we have special logging that is enabled specifically on the model side of the request. You may configure the runner access logs under the runners parameter at the top level of the bentofile.yaml
+Depending on how you've configured BentoML, the webserver may be separated from the model runner.
+In either case, we have special logging that is enabled specifically on the model side of the
+request. You may configure the runner access logs under the runners parameter at the top level of
+your ``bentofile.yaml``
```
Comment (parano) [inline docs/source/guides/logging.rst:86]:
```suggestion
your :code:`bentoml_configuration.yml`:
```

File: docs/source/guides/logging.rst [hunk 5]
```diff
@@ -2,94 +2,115 @@
...
+    import logging
+
+    ch = logging.StreamHandler()
+    ch.setLevel(DEBUG)
+
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+
+    bentoml_logger = logging.getLogger("bentoml")
+    bentoml_logger.addHandler(ch)
```
Comment (parano) [inline docs/source/guides/logging.rst:116]:
```suggestion
    import logging

    ch = logging.StreamHandler()
    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
    ch.setFormatter(formatter)

    bentoml_logger = logging.getLogger("bentoml")
    bentoml_logger.addHandler(ch)
    bentoml_logger.setLevel(logging.DEBUG)

```

Comment (parano) [reply docs/source/guides/logging.rst:116]: the sample code doesn't seem to work, calling setLevel on the bentoml logger fixed it for me

**Comment by sauyon** [comment]
Comments addressed.

**Review by aarnphm — APPROVED** [review]

**Pull request merged**